<launch>
  <node name="frcnn_detector" pkg="wrs_detector" type="detector_node.py" output="screen">
    <param name="detector" value="frcnn_default"/>
    <param name="mode" value="with_request"/>
    <param name="threshold" value="0.2"/>
    <param name="publish_image" value="true"/>

    <remap from="in" to="/hsrb/head_rgbd_sensor/rgb/image_rect_color" />
    <remap from="result" to="detection/bbox" />
    <remap from="result_image" to="detection/image" />
    <remap from="get_object_detection" to="detection/get_object_detection" />
  </node>
</launch>
